src/game/Tactical/OppList.cc

```diff
@@ -1375,16 +1375,14 @@ static INT16 ManLooksForMan(SOLDIERTYPE* pSoldier, SOLDIERTYPE* pOpponent, UINT8
 	if (!pOpponent->bActive || !pOpponent->bInSector || pOpponent->bLife <= 0 ||
 		pOpponent->sGridNo == NOWHERE )
 	{
-		SLOGE("ManLooksForMan - TARGET is inactive etc ID %d(%s)to ID %d",
-			pSoldier->ubID, pSoldier->name.c_str(), pOpponent->ubID);
 		return(FALSE);
 	}
 
 
 	// if he's looking for a guy who is on the same team
 	if (pSoldier->bTeam == pOpponent->bTeam)
 	{
-		SLOGE("ManLooksForMan - SAME TEAM ID %d(%s)to ID %d",
```

Comment:
Like I said in the issue, I can hit this also with the same soldier in both roles, so since this then returns false, we can add an independent `pSoldier == pOpponent` check right at the start.